test/regression_tests.sh

```diff
@@ -4,7 +4,7 @@ set -x
 #set -e
 
 TS_REPO="https://github.com/pytorch/serve"
-BRANCH=${1:-master}
+BRANCH="master"
```

Comment:
Why are we removing the option to provide a custom branch? It will be helpful to have this option if I want to run regression suite on a standalone basis

Reply:
This was buggy, we have already enhanced the script to take branch name as argument, else the default value will be "master"